apps/site/src/app/(home)/sections/Countdown/Countdown.tsx

```diff
@@ -0,0 +1,218 @@
+"use client";
+
+import bg_map from "@/assets/images/maps/countdown.svg";
+import Image from "next/image";
+import useWindow from "./useWindow";
+
+import styles from "./Countdown.module.scss";
+import CountdownClock from "./CountdownClock";
+import { useEffect, useState } from "react";
+
+import boat from "@/assets/icons/boat.png";
+
+interface CountdownProps {
+	schedule: {
+		title: string;
+		location?: string | undefined;
+		virtual?: string | undefined;
+		startTime: Date;
+		endTime: Date;
+		organization?: string | undefined;
+		hosts?: string[] | undefined;
+		description: any;
+	}[];
+}
+
+const Countdown: React.FC<CountdownProps> = ({ schedule }) => {
+	const hackStartTime = new Date("2024-11-02T10:00:00"); // TBD, zothacks start time
+	const hackEndTime = new Date("2024-11-02T22:00:00"); // TBD, zothacks end time
+
+	const [curTime, setCurTime] = useState(new Date());
+
+	useEffect(() => {
+		const i = setInterval(() => {
+			setCurTime(new Date());
+		}, 1000);
+
+		return () => clearInterval(i);
+	}, []);
+
+	const ended = schedule.filter((el) => el.endTime >= curTime);
+
+	const before =
+		ended.length > 0
+			? ended[0]
+			: {
+					title: "That's a wrap~",
+					location: "",
+					startTime: new Date(0),
+					endTime: new Date(0),
+				};
+
+	const after =
+		ended.length > 1
+			? ended[1]
+			: {
+					title: "That's a wrap~",
+					location: "",
+					startTime: new Date(0),
+					endTime: new Date(0),
+				};
+
+	const percentageCrossed =
+		before.endTime.getTime() > 0
+			? curTime.getTime() < before.startTime.getTime()
+				? 0
+				: ((before.endTime.getTime() - curTime.getTime()) /
```

Comment:
I mentioned this to you, but there is a chance this value is negated, moving the boat from end to beginning.

Subtracting this value from 100 might fix it, but there also might not be a problem to begin with. We will see in a bit.